client/utils/navigation.js

```diff
@@ -1,11 +1,6 @@
-import { NamedURLResolver } from 'react-router-named-routes'
-import { browserHistory } from 'react-router'
+import { history } from 'base'
 
 // eslint-disable-next-line import/prefer-default-export
-export function navigateTo(routeName) {
-  return browserHistory.push(namedRoute(routeName))
-}
-
-function namedRoute(name, params = {}) {
-  return NamedURLResolver.resolve(name, params)
+export function navigateTo(name) {
+  return history.push({ name })
```

Comment:
I think this can be `history.push(name)`; we don't need to wrap it in an object (though we can).